examples/form/mantine/useStepsForm/src/pages/posts/list.tsx

```diff
@@ -0,0 +1,49 @@
+import { useTable } from "@pankod/refine-core";
+import { EditButton, List, Pagination, Table } from "@pankod/refine-mantine";
+
+import { IPost } from "../../interfaces";
+
+export const PostList: React.FC = () => {
+    const { tableQueryResult, current, setCurrent, pageCount } =
+        useTable<IPost>({
+            resource: "posts",
+        });
```

Comment:
```suggestion
        useTable<IPost>();
```
is it necessary to add a resource?  if it is you can skip it.